.github/workflows/build-triton-wheel.yml

```diff
@@ -9,11 +9,13 @@ on:
       - .github/workflows/build-triton-wheel.yml
       - .github/scripts/build_triton_wheel.py
       - .github/ci_commit_pins/triton.txt
+      - .github/ci_commit_pins/triton-rocm.txt
   pull_request:
     paths:
       - .github/workflows/build-triton-wheel.yml
       - .github/scripts/build_triton_wheel.py
       - .github/ci_commit_pins/triton.txt
+      - .github/ci_commit_pins/triton-rocm.txt
```

Comment:
@malfet @dllehr-amd Might be just redundant and not a bigger issue, but I believe this will trigger both ROCm and CUDA triton wheel builds even if only one of the commit hashes changed